nemo/collections/asr/parts/numba/rnnt_loss/utils/cuda_utils/gpu_rnnt_kernel.py

```diff
@@ -961,7 +961,9 @@ def compute_tdt_alphas_kernel(
             if t > 0 and t < T:
```

Comment:
Hmm, it seems it's customary when writing CUDA kernel code to explicitly provide dimensions of tensors in the arguments (e.g. the maxT, maxU, vocabulary_size params are all passed into those functions). I haven't looked into this in detail but my guess is, those "tensor" parameters passed in functions are merely just GPU memory locations and might not be able to provide dimension information. 

Even if this is not the case, maybe let's do this in another PR since it's not of the same scope of this change?